data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/illuminates/illuminates_negative_min_value.cfg

```diff
@@ -0,0 +1,33 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [illuminates]min_value=
+##
+# Actions:
+# Give all units 50% self-illuminates bonus damage with a min_value=-10
+# Attack each other
+##
+# Expected end state:
+# min_value does nothing
```

Comment:
It is having an effect; or is there another explanation for the expected value being 90 instead of 50?

Reply:
It's having an effect, this is just copy/paste error.

Reply:
Removed.